src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -132,56 +131,79 @@ public Response getRtFeeds() {
     private static HashMap<String, ScheduledExecutorService> runningTasks = new HashMap<>();
 
     @PUT
-    @Path("/{id}/{updateInterval}/monitor")
-    public Response getID(@PathParam("id") int id, @PathParam("updateInterval") int updateInterval) {
+    @Path("/{id}/{clientId}/{updateInterval}/monitor")
```

Comment:
Given that this is an HTTP Put and modifies the database and starts an action  (monitoring), this should probably be named "startMonitor" instead of "getID".